include/seastar/core/sstring.hh

```diff
@@ -36,6 +36,7 @@
 #include <type_traits>
 #include <seastar/util/std-compat.hh>
 #include <seastar/core/temporary_buffer.hh>
+#include <boost/lexical_cast.hpp>
```

Comment:
Why do you need to include that header? It sounds heavy (judging by the "boost" prefix alone), and it's included in a rather common Seastar header, which may negatively impact compilation times. And it doesn't look like you actually rely on any contents of this header here. If you need it for the unit test, it can land right in sstring_test.cc

Reply:
@avelanarius ^ this should be fixed before this can be considered for merge

Reply:
@avelanarius please fix and force push a new version